src/pyfuse3/__init__.pyi

```diff
@@ -9,7 +9,15 @@ This file is part of pyfuse3. This work may be distributed under
 the terms of the GNU LGPL.
 '''
 
-from ._pyfuse3 import Operations, async_wrapper, FileHandleT, FileNameT, FlagT, InodeT, ModeT
+# re-exports
+from ._pyfuse3 import (
+    Operations as Operations,
+    FileHandleT as FileHandleT,
+    FileNameT as FileNameT,
+    FlagT as FlagT,
+    InodeT as InodeT,
+    ModeT as ModeT
+)
```

Comment:
Yet I only used "from ... import ... as ..." when needing to rename.
So just importing these here is not sufficient?